setup/sql/dbupdate_custom_renameme.php

```diff
@@ -0,0 +1,7 @@
+<#1>
+<?php
+	if($ilDB->tableExists('svy_qst_oblig'))
+		$ilDB->dropTable('svy_qst_oblig');
+	if($ilDB->tableExists('svy_qst_oblig_seq'))
+		$ilDB->dropTable('svy_qst_oblig_seq');
+?>
```

Comment:
In general you can name these files "dbupdate_custom.php" in the branch (so they work directly in the setup). I will remove them when merging.